Formula/wiremock-standalone.rb

```diff
@@ -0,0 +1,19 @@
+class WiremockStandalone < Formula
+  desc "Simulator for HTTP-based APIs"
+  homepage "http://wiremock.org"
+  url "https://search.maven.org/remotecontent?filepath=com/github/tomakehurst/wiremock-standalone/2.12.0/wiremock-standalone-2.12.0.jar"
+  sha256 "488577b1afb193315afa25a9e2577c54fba8a893a7887895672f7317add454f8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "wiremock-standalone-#{version}.jar"
+    bin.write_jar_script libexec/"wiremock-standalone-#{version}.jar", "wiremock"
+  end
+
+  test do
+    system "#{bin}/wiremock", "--help"
```

Comment:
This will need to be some form of functionality testing rather than simply checking the `--help` output, I suspect.

Reply:
I thought about doing a more comprehensive check consisting on launching a fake server with a very basic REST API, doing a curl request to the fake server and checking if the response is as expected.

To launch a WireMock fake server, I need a port. If I use a fixed port, the formula check will crash if the port is taken on the computer where the formula will be installed. This makes the installation process non-repeatable.

Alternatively, I could search for a free random port. I couldn't find any easy way to achieve it. @DomT4 if you have any suggestion, is more than welcome.

Reply:
You can get a valid port number with:

```
    require "socket"

    server = TCPServer.new(0)
    port = server.addr[1]
    server.close
```